homeassistant/components/opentherm_gw/sensor.py

```diff
@@ -1,20 +1,621 @@
+            friendly_name_format="Total Burner Hours {}",
+            device_class=SensorDeviceClass.DURATION,
+            state_class=SensorStateClass.TOTAL,
+            native_unit_of_measurement=UnitOfTime.HOURS,
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_CH_PUMP_HOURS,
+            friendly_name_format="Central Heating Pump Hours {}",
+            device_class=SensorDeviceClass.DURATION,
+            state_class=SensorStateClass.TOTAL,
+            native_unit_of_measurement=UnitOfTime.HOURS,
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_DHW_PUMP_HOURS,
+            friendly_name_format="Hot Water Pump Hours {}",
+            device_class=SensorDeviceClass.DURATION,
+            state_class=SensorStateClass.TOTAL,
+            native_unit_of_measurement=UnitOfTime.HOURS,
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_DHW_BURNER_HOURS,
+            friendly_name_format="Hot Water Burner Hours {}",
+            device_class=SensorDeviceClass.DURATION,
+            state_class=SensorStateClass.TOTAL,
+            native_unit_of_measurement=UnitOfTime.HOURS,
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_MASTER_OT_VERSION,
+            friendly_name_format="Thermostat OpenTherm Version {}",
+            suggested_display_precision=SENSOR_FLOAT_SUGGESTED_DISPLAY_PRECISION,
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_SLAVE_OT_VERSION,
+            friendly_name_format="Boiler OpenTherm Version {}",
+            suggested_display_precision=SENSOR_FLOAT_SUGGESTED_DISPLAY_PRECISION,
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_MASTER_PRODUCT_TYPE,
+            friendly_name_format="Thermostat Product Type {}",
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_MASTER_PRODUCT_VERSION,
+            friendly_name_format="Thermostat Product Version {}",
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_SLAVE_PRODUCT_TYPE,
+            friendly_name_format="Boiler Product Type {}",
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_SLAVE_PRODUCT_VERSION,
+            friendly_name_format="Boiler Product Version {}",
+        ),
+    ),
+    (
```

Comment:
These look static and don't belong in a sensor tbh

Reply:
They could be used to populate the device info when splitting off the thermostat and boiler into separate devices, but in my case all of these are `unknown` as support is not mandatory for manufacturers.